src/pages/AboutUs/OurTeam/index.jsx

```diff
@@ -1,9 +1,48 @@
 import styles from "./ourteam.module.css";
+import Navbar from "../../Navbar";
+import Footer from "../../Footer";
 
 const OurTeam = () => {
   return (
     <div className={styles.OurTeam}>
-      <p>Our Team Page</p>
+      <Navbar />
+      <div className={styles.body}>
```

Comment:
Seems like this segment is still hard-coded. Not sure if this will integrate with Notion also, but I think can be fixed before merging.